src/platform/forms-system/test/js/review/ArrayField.unit.spec.jsx

```diff
@@ -259,9 +321,7 @@ describe('Schemaform review <ArrayField>', () => {
     const uiSchema = {
       'ui:title': 'List of things',
       items: {
-        test: {
-          type: 'string',
-        },
+        test: {},
```

Comment:
Is this change pertinent to the PR?

Reply:
Oops, nope... but `type` usually isn't set in a `uiSchema`